002-quarkus-all-extensions/pom.xml

```diff
@@ -470,32 +397,31 @@
         <!--            <groupId>io.quarkus</groupId>-->
         <!--            <artifactId>quarkus-hibernate-reactive</artifactId>-->
         <!--        </dependency>-->
-        <dependency>
-            <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-jdbc-db2</artifactId>
-        </dependency>
-        <!--        OIDC / Keycloak stuff disabled as it would require running server-->
         <!--        <dependency>-->
         <!--            <groupId>io.quarkus</groupId>-->
-        <!--            <artifactId>quarkus-keycloak-admin-client</artifactId>-->
+        <!--            <artifactId>quarkus-jdbc-db2</artifactId>-->
+        <!--        </dependency>-->
+        <!--        <dependency>-->
+        <!--            <groupId>io.quarkus</groupId>-->
+        <!--            <artifactId>quarkus-reactive-db2-client</artifactId>-->
         <!--        </dependency>-->
         <dependency>
             <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-kubernetes-config</artifactId>
+            <artifactId>quarkus-keycloak-admin-resteasy-client</artifactId>
         </dependency>
         <dependency>
             <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-minikube</artifactId>
+            <artifactId>quarkus-kubernetes-config</artifactId>
         </dependency>
-        <!--  We cannot have picocli and REST assured at the same time.
         <dependency>
             <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-picocli</artifactId>
-        </dependency> -->
-        <dependency>
-            <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-reactive-db2-client</artifactId>
+            <artifactId>quarkus-minikube</artifactId>
         </dependency>
+        <!--  We cannot have picocli and REST assured at the same time. -->
```

Comment:
that's hard to believe, why?

Reply:
FYI we do use REST Assured with Picocli in the QE TS.

Reply:
Probably result of missing Picocli command. That sounds valid, let's drop it or add a command.